src/css/blog.css

```
@@ -0,0 +1,491 @@
+   The BlogLayout swizzle also injects data-paper-page, which
+   activates navbar / footer / accent rules in custom.css.
+   All colour values reference CSS variables from tokens.css.
+   ============================================================ */
+
+/* ── Page shell ─────────────────────────────────────────── */
+/* Beats [data-theme='dark'] body (specificity 0,1,1) */
+html.blog-wrapper body,
+html.blog-wrapper .main-wrapper {
+  background-color: var(--paper-bg) !important;
+  color: var(--paper-text-body);
+  font-family: 'Geist', -apple-system, sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)